quiz_app/user/templates/login.html

```diff
@@ -0,0 +1,29 @@
+{% if form.errors %}
```

Comment:
Django offers built-in out-of-the box login and register template.
It's good that you created your own, commenting just co we know about it.